src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/domain/CaseValidationResponse.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sscs.bulkscancore.domain;
+
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class CaseValidationResponse {
```

Comment:
Isn't this what you want?
https://github.com/hmcts/core-case-data-store-client/blob/master/src/main/java/uk/gov/hmcts/reform/ccd/client/model/AboutToStartOrSubmitCallbackResponse.java